File: sem5/Web-программирование/lab1/src/com/verenich/entities2d/Point2D.java
package com.verenich.entities2d;

public class Point2D {
	private double m_x;
	private double m_y;
	
	/**
	 * Constructs the Point2D object
	 * @param x -- x-coordinate
	 * @param y -- y-coordinate
	 */
	public Point2D(double x, double y)
	{
		this.m_x = x;
		this.m_y = y;
	}
	
	public void setX(double x)
	{
		this.m_x = x;
	}
	
	public void setY(double y)
	{
		this.m_y = y;
	}
	
	public double getX()
	{
		return m_x;
	}
	
	public double getY()
	{
		return m_y;
	}
	
	/**
     * @return a string representing a Point2D
     */
    @Override
	public String toString() {
        return "Point2D{ " + m_x + ", " + m_y + " }";
    }
	
}

File: sem1/projects C++/Project1/Project1/Point2D.cpp
#include "Point2D.h"

double Point2D::getPhi()
{
	return 0.0;
}

Point2D::Point2D(double aX, double aY)
{
	setX(aX);
	setY(aY);
}

std::string toString() {
	return "(";
}


double Point2D::getX() const {
	return x;
}


void Point2D::setX(double aX) {
	x = aX;
}


double Point2D::getY() const {
	return y;
}


void Point2D::setY(double aY) {
	y = aY;
}

double Point2D::getPhi() const {
	if (x == 0 && y == 0) {
		std::cout << "Epic fail!\n";
	}
}


Point2D::~Point2D(void) {

}